### PR TITLE
[FEATURE] Add atomic renderer for `ExpectMulticolumnSumToEqual`

### PR DESCRIPTION
**Pull request by NathanFarmer**
Add missing renderer using existing pattern

- [x] Description of PR changes above includes a link to [an existing GitHub issue](https://github.com/great-expectations/great_expectations/issues)
- [x] PR title is prefixed with one of: [BUGFIX], [FEATURE], [DOCS], [MAINTENANCE], [CONTRIB]
- [x] Code is linted - run `invoke lint` (uses `ruff format` + `ruff check`)
- [x] Appropriate tests and docs have been updated

### CODE REVIEW

File: great_expectations/expectations/core/expect_multicolumn_sum_to_equal.py [hunk 1]
```diff
@@ -227,6 +232,49 @@ def schema_extra(schema: Dict[str, Any], model: Type[ExpectMulticolumnSumToEqual
                 }
             )
 
+    @classmethod
+    def _prescriptive_template(
```
Comment (NathanFarmer) [inline great_expectations/expectations/core/expect_multicolumn_sum_to_equal.py:236]: Code coverage is apparently poor here, but I believe this is now a legacy pattern anyway.

File: great_expectations/expectations/core/expect_multicolumn_sum_to_equal.py [hunk 2]
```diff
@@ -227,6 +232,49 @@ def schema_extra(schema: Dict[str, Any], model: Type[ExpectMulticolumnSumToEqual
...
+                renderer_configuration=renderer_configuration,
+            )
+
+        if params.mostly and params.mostly.value < 1.0:
+            renderer_configuration = cls._add_mostly_pct_param(
+                renderer_configuration=renderer_configuration
+            )
+            template_str += " must be $sum_total, at least $mostly_pct % of the time."
+        else:
+            template_str += " must be $sum_total."
```
Comment (Kilo59) [inline great_expectations/expectations/core/expect_multicolumn_sum_to_equal.py:272]: `non-blocking`
You could accumulate these items in a list and then `.join()` them at the end.
String operations in python are actually relatively fast though, so there might not actually be much benefit to this. 🤷 
```python
template_parts: list[str] = []
if params.column_list:
  array_param_name = "column_list"
  param_prefix = "column_list_"
  renderer_configuration = cls._add_array_params(
    array_param_name=array_param_name,
    param_prefix=param_prefix,
    renderer_configuration=renderer_configuration,
  )
  template_parts += "Sum across columns "
  template_parts += cls._get_array_string(
    array_param_name=array_param_name,
    param_prefix=param_prefix,
    renderer_configuration=renderer_configuration,
  )

...

  renderer_configuration.template_str = "".join(template_parts)
  return renderer_configuration
```

**Review by Kilo59 — APPROVED** [review]
LGTM.

~Do we usually test these?~ From your comment, it looks like no.

File: great_expectations/expectations/core/expect_multicolumn_sum_to_equal.py (hunk 2 above)
Comment (NathanFarmer) [reply great_expectations/expectations/core/expect_multicolumn_sum_to_equal.py:272]: Prefer to keep it the same as it is on every other core Expectation. My understanding is there is a new (simpler) rendering pattern, and we may be re-evaluating the entire rendering arch.

File: great_expectations/expectations/core/expect_multicolumn_sum_to_equal.py (hunk 1 above)
Comment (NathanFarmer) [reply great_expectations/expectations/core/expect_multicolumn_sum_to_equal.py:236]: There are tests for the end result, but it looks like we missed unit tests for the helpers.

**Pull request merged**